meta-ostro/recipes-swupd/swupd-client/swupd-client_%.bbappend

```diff
@@ -23,16 +23,9 @@ SYSTEMD_SERVICE_${PN} += "${@ 'efi-combo-trigger.service' if ${OSTRO_USE_DSK_IMA
 # And activate it.
 SYSTEMD_AUTO_ENABLE_${PN} = "enable"
 
-do_compile_append() {
-    if [ "${OSTRO_USE_DSK_IMAGES}" = "True" ]; then
-        ${CC} ${LDFLAGS} ${WORKDIR}/efi_combo_updater.c  -Os -o ${B}/efi_combo_updater `pkg-config --cflags --libs glib-2.0`
-    fi
-}
-
 do_install_append () {
     if [ "${OSTRO_USE_DSK_IMAGES}" = "True" ]; then
-        install -d ${D}/usr/bin
-        install ${B}/efi_combo_updater ${D}/usr/bin/
+        install -m 0744 ${WORKDIR}/efi_combo_updater.sh ${D}/usr/bin/
```

Comment:
@igor-stoppa,  I think file name mismatch. so either 
update 'ExecStart=' in efi-combo-trigger.service to point to script.
or change the script name
install -m 0744 ${WORKDIR}/efi_combo_updater.sh ${D}/usr/bin/efi_combo_updater